spec/config_spec.rb

```diff
@@ -27,4 +33,63 @@ module Builderator
       expect(Config.build_name).to eq 'builderator'
     end
   end
+
+  RSpec.describe Config, '#compile' do
+    before(:example) do
+      Builderator::Config::GLOBAL_DEFAULTS.reset!
+      Builderator::Config.reset!
+    end
+
+    10.times do |itr|
```

Comment:
Actually, the loops are probably not needed anymore. The issue that they were testing was caused by a `.shuffle` call in one of the test resources... Anyone disagree?